### PR TITLE
Added Gripper

### PR DESCRIPTION
**Pull request by awesomeatpiano**
cmds, buttons, and subsystem

### CODE REVIEW

**Review by mastermind-mayhem — APPROVED** [review]

File: src/main/java/frc/robot/subsystems/GripperSubsystem.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  }
+
+  public void rotateinGripper() {
+
+    gripmot.set(0.33);
+  }
+
+  public void stopGripper() {
+
+    gripmot.set(0);
```
Comment (mastermind-mayhem) [inline src/main/java/frc/robot/subsystems/GripperSubsystem.java:30]: change to "gripmot.stopmotor();"

**Review by mastermind-mayhem — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/Robot.java [hunk 2]
```diff
@@ -56,7 +56,7 @@ public void disabledPeriodic() {}
   /** This autonomous runs the autonomous command selected by your {@link RobotContainer} class. */
   @Override
   public void autonomousInit() {
-    m_robotContainer.clawbrake();
+    // m_robotContainer.clawbrake();
```
Comment (mathyourlife) [inline src/main/java/frc/robot/Robot.java:59]: delete instead of comment out

File: src/main/java/frc/robot/RobotContainer.java [hunk 3]
```diff
@@ -79,11 +80,14 @@ public class RobotContainer {
...
-  private final ClampCmd clampCmd = new ClampCmd(clawSubsystem);
-  private final ReleaseCmd releaseCmd = new ReleaseCmd(clawSubsystem);
-  private final ShiftLeftCmd shiftLeftCmd = new ShiftLeftCmd(clawSubsystem);
-  private final ShiftRightCmd shiftRightCmd = new ShiftRightCmd(clawSubsystem);
-  private final StopClawCmd stopClawCmd = new StopClawCmd(clawSubsystem);
+  // private final ClampCmd clampCmd = new ClampCmd(clawSubsystem);
+  // private final ReleaseCmd releaseCmd = new ReleaseCmd(clawSubsystem);
+  // private final ShiftLeftCmd shiftLeftCmd = new ShiftLeftCmd(clawSubsystem);
+  // private final ShiftRightCmd shiftRightCmd = new ShiftRightCmd(clawSubsystem);
+  // private final StopClawCmd stopClawCmd = new StopClawCmd(clawSubsystem);
```
Comment (mathyourlife) [inline src/main/java/frc/robot/RobotContainer.java:87]: delete instead of comment out

File: src/main/java/frc/robot/RobotContainer.java [hunk 4]
```diff
@@ -93,9 +97,9 @@ public void setcoast() {
     drivetrain.setCoast();
   }
 
-  public void clawbrake() {
-    clawSubsystem.clawbrake();
-  }
+  // public void clawbrake() {
+  //   clawSubsystem.clawbrake();
+  // }
```
Comment (mathyourlife) [inline src/main/java/frc/robot/RobotContainer.java:102]: D.I.O.C.O

File: src/main/java/frc/robot/subsystems/GripperSubsystem.java [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class GripperSubsystem extends SubsystemBase {
+  /** Creates a new GripperSubsystem. */
+  public GripperSubsystem() {}
+
+  private final CANSparkMax gripmot = new CANSparkMax(Constants.Gripper, MotorType.kBrushless);
```
Comment (mathyourlife) [inline src/main/java/frc/robot/subsystems/GripperSubsystem.java:16]: yay, brushless :)

**Review by mastermind-mayhem — APPROVED** [review]

**Pull request merged**